.external-pins/docker/scout-sbom-indexer___1

```diff
@@ -1 +1 @@
-sha256:254437e05256f393fe4b1b44e2811ec6a05ad7e67baacf70d49552af7c6ac318
+sha256:cc1418c9ab380cdf4b792660a8c3b4ff2759731468a49bc0d7ad5806b1fe20f8
```

Comment:
It will be `1-doi` when 1.6.5 is released

Reply:
If we leave the filename as-is, then the automation (https://doi-janky.infosiftr.net/job/update.sh/job/external-pins/) will continue to try to point it at the newest `:1` image. It'll create a branch in docker-library-bot's fork of `official-images` (where all our automated `update.sh` PR branches are).

`docker/scout-sbom-indexer:5048f80-doi` isn't a great tag to follow since we'll have to update it once `1-doi` is published, but it makes it clearer where the digest comes from (and the scripts in `.external-pins` will continue to be correct).

Reply:
Would it be simpler to wait for a release and renamed it `scout-sbom-indexer___1-doi`?

Reply:
Yes, and it looks like we no longer have to wait since it is released already: https://explore.ggcr.dev/?image=docker%2Fscout-sbom-indexer:1-doi